src/RestApi/StoreApi/README.md

```diff
@@ -722,66 +753,6 @@ Example response:
 }
 ```
 
-### Edit single cart item
-
-Edit an item in the cart.
-
-```http
-PUT /cart/items/:key
-```
-
-| Attribute  | Type    | Required | Description                        |
-| :--------- | :------ | :------: | :--------------------------------- |
-| `key`      | string  |   Yes    | The key of the cart item to edit.  |
-| `quantity` | integer |   Yes    | Quantity of this item in the cart. |
-
-```http
-curl --request PUT https://example-store.com/wp-json/wc/store/cart/items/3ef815416f775098fe977004015c6193&quantity=10
```

Comment:
Note: I've removed these `curl` examples when moving docs because they don't work (e.g. endpoint path is a param in WP rest API). @mikejolley what do you think about removing the curl examples throughout? Let me know if they work for you.

Reply:
Can we not just fix the examples if you see a problem? Otherwise we cannot document a request and response.

Reply:
Sure - I'll get these working and post and update. I definitely think it's a nice idea to have pasteable examples.

Just realised why these aren't working for me – I had plain permalinks option set. I'll add a reminder about that (gotcha) to the doc.